scripts/prepare_sector_network.py

```diff
@@ -19,13 +19,17 @@
 from networkx.algorithms.connectivity.edge_augmentation import k_edge_augmentation
 from networkx.algorithms import complement
 from pypsa.geo import haversine_pts
+from pypsa.io import import_components_from_dataframe
 
 import logging
 logger = logging.getLogger(__name__)
 
 from types import SimpleNamespace
 spatial = SimpleNamespace()
 
+from distutils.version import LooseVersion
```

Comment:
This is now deprecated. Use packaging tools instead:

https://github.com/PyPSA/PyPSA/blob/6bf79d8f7a7e01c6cac6668dc0675f33fa90b29b/pypsa/networkclustering.py#L23


https://github.com/PyPSA/PyPSA/blob/6bf79d8f7a7e01c6cac6668dc0675f33fa90b29b/pypsa/networkclustering.py#L744

Reply:
thanks, I changed it now!